http/http_connection.hpp

```diff
@@ -300,10 +293,6 @@ class Connection :
                                         const boost::system::error_code& ec) {
                                         if (ec)
                                         {
-                                            BMCWEB_LOG_ERROR
```

Comment:
Hmm, was this a useful error that we should keep?

Reply:
It didn't hurt so added back. Done.